### PR TITLE
Feat: Added bundle size visualizer for rollup

### PR DESCRIPTION
**Pull request by saikumarrs**
## Description of the change

Added bundle visualizer (for all SDKs) for rollup.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

N/A

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/534)

### CODE REVIEW

File: rollup.intgConfig.js [hunk 1]
```diff
@@ -108,7 +108,23 @@ export default {
...
+              edge: "15",
+              firefox: "40",
+              ie: "10",
+              chrome: "37",
+              safari: "7",
+              opera: "23",
+            },
+          },
+        ],
+      ],
```
Comment (MoumitaM) [inline rollup.intgConfig.js:127]: This doesn’t have any benefit unless you are importing core-js to your project. [Ref](https://babeljs.io/docs/en/babel-preset-env#usebuiltins-entry)

**Review by MoumitaM — CHANGES_REQUESTED** [review]

File: rollup.intgConfig.js (hunk 1 above)
Comment (saikumarrs) [reply rollup.intgConfig.js:127]: True. It's a preemptive configuration.

Comment (MoumitaM) [reply rollup.intgConfig.js:127]: Let’s remove it then.

Comment (saikumarrs) [reply rollup.intgConfig.js:127]: This is added to serve as a source of truth for browser support. We might have to update the values after final testing.

**Pull request merged**